apps/bfd-model/bfd-model-rif-samples/dev/synthea-test-plan.md

```diff
@@ -0,0 +1,235 @@
+    );
+    
+    DELETE FROM public.inpatient_claim_lines
+        WHERE "clm_id" in (SELECT "clm_id" FROM public.inpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999'
+    );
+    
+    DELETE FROM public.outpatient_claim_lines
+        WHERE "clm_id" in (SELECT "clm_id" FROM public.outpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999'
+    );
+    
+    DELETE FROM public.snf_claim_lines
+        WHERE "clm_id" in (SELECT "clm_id" FROM public.snf_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999'
+    );
+        
+        
+    DELETE FROM public.carrier_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.dme_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.hha_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    
+    DELETE FROM public.hospice_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.inpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.outpatient_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.snf_claims
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    
+    DELETE FROM public.partd_events
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiary_monthly
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries_history
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';
+        
+    DELETE FROM public.beneficiaries
+    WHERE "bene_id" >='-10000000000000'
+        AND "bene_id" <='-10000000009999';`
+
+
+## Load Synthetic Data in Hosted Environments
+
+1. We need to make sure in TEST, SBX or PROD there are no collisions for the main properties in synthetic data i.e. bene_id, mbi_num, claim_id, etc. Before loading data, agregate RIF data to generate the following queries: 
+
+        Select count(*) from public.beneficiaries where mbi_num in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where bene_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.beneficiaries where hicn_unhashed in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.partd_events where pde_id in (‘XXX’,’YYY’,…);
+
+
+        -- For all claim types
+
+        Select count(*) from public.carrier_claims where claim_id in (‘XXX’,’YYY’,…);
+
+        Select count(*) from public.carrier_claims where clm_grp_id BETWEEN ‘XXX’ AND ’YYY’;
```

Comment:
I think we are missing some queries here for the carrier cntrl num and the fi_doc_num, possibly other things (check the end_state.properties)